packages/hawtio/src/plugins/camel/endpoints/endpoints-service.ts

```diff
@@ -280,3 +292,13 @@ function parseMessagesFromXml(pDoc: XMLDocument): MessageData[] {
 
   return messagesData
 }
+
+export async function getEndpointStatistics(node: MBeanNode) {
+  let stats: EndpointStatistics[] = []
+  const registry = getDefaultRuntimeEndpointRegistry(node)
+  if (registry) {
+    const res = await jolokiaService.execute(registry.objectName ?? '', 'endpointStatistics()')
```

Comment:
if `objectName` is absent it doesn't make sense to invoke the method. Rather shouldn't we guard it based on its presence and if it's absent something else should be done, like throwing or logging error?

Reply:
updated